src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultAction.tsx

```diff
@@ -231,10 +231,15 @@ const DefaultAction = ({
     isSmiteAction: new Decimal(reputationChange).isNegative(),
     removedSafes,
     removedSafesString,
+    safeAddress: addedSafe && (
+      <MaskedAddress address={addedSafe.contractAddress} />
+    ),
+    selectedChain: addedSafe && GNOSIS_SAFE_NAMES_MAP[addedSafe.chainId],
```

Comment:
A nitpick again but I would go for a more precise name, something like `chainName` (name is an important part here). But up to you...